support/slack.js

```diff
@@ -57,8 +57,8 @@ async function inviteUser(token, channel, user) {
 async function describeUser(token, user) {
   return request({
     url: 'https://slack.com/api/users.info',
-    method: 'POST',
-    form: { token, user },
```

Comment:
Wait, what?! xD

Reply:
Yeah was this ever used before? No way that could’ve worked

Reply:
It would. In a wrong way of obtaining "GET" via "POST". :D

Reply:
Well I mean it wouldn’t have errored but it also wouldn’t have produced the desired outcome

Reply:
This was used in automod code and it's working now. The slack API has preferred methods rather than required methods so they do some translations and reply as requested. Also, even with the `POST`, I was getting a valid response.